client/src/controllers/KeyboardController.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller } from '@hotwired/stimulus';
+import Mousetrap from 'mousetrap';
+
```

Comment:
```suggestion

/**
 * Adds the ability to trigger a button click event using a keyboard shortcut
 * declared on the controlled element.
 *
 * @example
 * ```html
 * <button type="button" data-controller="w-kbd" data-w-kbd-key="[">Trigger me with the <kbd>[</kbd> key.</button>
 * ```
 */
```

I'd recommend we add a simple JSDoc intro with a basic example.